mesonbuild/minstall.py

```diff
@@ -532,6 +531,23 @@ def install_targets(self, d):
                     else:
                         raise
 
+def rebuild_all(wd: str) -> bool:
```

Comment:
`Path` and `environment` are undefined, they aren't imported in `minstall.py` (as they are in `mintro.py`). 
This fails `meson install`.

https://github.com/mesonbuild/meson/blob/2f836e3accd112a52bb8d3ef5d44b7e0fb692fcf/mesonbuild/minstall.py#L534

Made a PR #8136

Reply:
oops, thanks.